resources/skins/nudus/html_wrapper.html

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8" ?>

```

Comment:
Btw, this line is redundant, just make sure that resulting page is HTML5-compliant.

https://www.w3schools.com/tags/tag_doctype.asp

Reply:
Oh, I thought it's there by design.

Reply:
It makes no harm, just was pointing out that HTML5 stuff.

Reply:
Leave it open for now.

Reply:
Removed.